aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -112,13 +112,13 @@
    * switch from uploading using a single put object request to uploading using multipart upload
    * (default: 1.5).
    */
-  public static final String S3FILEIO_MULTIPART_THRESHOLD_FACTOR = "s3fileio.multipart.threshold";
+  public static final String S3FILEIO_MULTIPART_THRESHOLD_FACTOR = "s3.multipart.threshold";
```

Comment:
wouldn't this be `s3.multipart-threshold`?

Reply:
My understanding is that `s3.multipart.*` has all the multipart upload configs, that's why I kept it unchanged.

Reply:
yup, that makes sense